src/amchart/CommonFunnel.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 (function(root, factory) {
     if (typeof define === "function" && define.amd) {
-        define(["d3", "../common/HTMLWidget", "amcharts.funnel", "require"], factory);
+        define(["d3", "../common/HTMLWidget", "amcharts.funnel", "require", "amcharts.plugins.export"], factory);
     } else {
         root.amchart_CommonFunnel = factory(root.d3, root.common_HTMLWidget, root.AmCharts, root.require);
```

Comment:
in the non amd world it will get loaded into window.AmCharts which happens to be root i believe when non amd is loaded? correct me if i am wrong here but i cant just overwrite root.AmCharts again afterloading amcharts.funnel into it from within here?